website/content/en/preview/concepts/setup.md

```diff
@@ -0,0 +1,551 @@
+---
+title: "Karpenter CloudFormation Setup"
+linkTitle: "Karpenter CloudFormation Setup"
+weight: 5
+description: >
+  Descriptions of how Karpenter uses CloudFormation to set up permissions
+---
+When you create a cluster to use with Karpenter in the [Getting Started with Karpenter]({{< relref "../getting-started/" >}}) guide, the procedure uses CloudFormation to prepare the cluster for Karpenter to be able to create and manage nodes, as well as gather and respond to interruption events.
+This document describes the `cloudformation.yaml` file used in that guide.
+These descriptions will be useful to understand:
+
+* What Karpenter is authorized to do with your EKS cluster and AWS resources when you use the `cloudformation.yaml` file
+* What permissions you need to set up if you are adding Karpenter to an existing cluster
+
+# Review the cloudformation.yaml file
+
+To download a particular version of `cloudformation.yaml`, set the version and use `curl` to pull the file to your local system:
+
+```bash
+export KARPENTER_VERSION=v0.29.0
+curl https://raw.githubusercontent.com/aws/karpenter/"${KARPENTER_VERSION}"/website/content/en/preview/getting-started/getting-started-with-karpenter/cloudformation.yaml > cloudformation.yaml
+```
+
+The `cloudformation.yaml` file starts with the following information:
+
+```
+AWSTemplateFormatVersion: "2010-09-09"
+Description: Resources used by https://github.com/aws/karpenter
+Parameters:
+  ClusterName:
+    Type: String
+    Description: "EKS cluster name"
+Resources:
+```
+
+The rest of the `cloudformation.yaml` file describes the resources that CloudFormation deploys.
+The sections of that file can be grouped together under the following general headings:
+
+* **Node Authorization**: Creates a NodeInstanceProfile, attaches a NodeRole to it, and connects it to an IAM Identity Mapping that Karpenter uses. This defines the permissions each node managed by Karpenter has to access EC2 and other AWS resources.
+* **Karpenter Controller Authorization**:  Creates a service account (named `karpenter`) that is combined with the KarpenterControllerPolicy. The KarpenterControllerPolicy allows the Karpenter controller to control assets within EC2 specifically and within AWS in general.
+* **Interruption Handling**: The interruption handling sections of this file allow the Karpenter controller to see and respond to interruptions that occur with the nodes that Karpenter is managing. Interruptions can reflect things like a spot instance going away or physical hardware going down. Allowing the Karpenter controller to see these interruptions allows Karpenter to respond by bringing nodes up and down and moving workloads.
+
+A lot of the object naming that is done by `cloudformation.yaml` is based on the following:
+
+* Cluster name: With a user name of `joe` the Getting Started Guide would name your cluster `joe-karpenter-demo`
```

Comment:
```suggestion
* Cluster name: With a user name of `joe` the Getting Started Guide would name your cluster `joe-karpenter-demo`
```

Just curious: Is there a typical name that AWS docs typically use as generic usernames? Some good examples for common names in the security space (for example) are "Bob" and "Alice"

Reply:
Honestly, I don't know. But bob is fine with me. I'll use that.

Reply:
Done